crates/viewer/re_viewer/src/screenshotter.rs

```diff
@@ -89,7 +89,7 @@ impl Screenshotter {
             let h = image.height() as _;
             let image =
                 image::RgbaImage::from_raw(w, h, bytemuck::pod_collect_to_vec(&image.pixels))
-                    .expect("Failed to create image");
+                    .expect("Failed to create image (internal error)");
```

Comment:
I like the "internal error" message for our users, but I think we should add that to the panic handler of `re_crash_handler` instead, since any panic is an internal error.

Reply:
I would agree in most but not all cases, I think this points to a counter-example as I commented below: https://github.com/rerun-io/rerun/pull/8745/files#r1928954634

Reply:
How do you think we should proceed with this?

Reply:
A panic _should_ only happen because of a bug in the rerun codebase, which is what I would call an internal error. If there are panics that could happen because of user error, we should fix that separatedly.